apisix/plugins/clickhouse-logger.lua

```diff
@@ -72,11 +74,12 @@ end
 local function send_http_data(conf, log_message)
     local err_msg
     local res = true
-    local url_decoded = url.parse(conf.endpoint_addr)
+    local selected_endpoint_addr = conf.endpoint_addr[math_random(#conf.endpoint_addr)]
+    local url_decoded = url.parse(selected_endpoint_addr)
     local host = url_decoded.host
     local port = url_decoded.port
 
-    core.log.info("sending a batch logs to ", conf.endpoint_addr)
+    core.log.info("sending a batch logs to ", selected_endpoint_addr)
```

Comment:
if `selected_endpoint_addr ` is a table. would it better to use `core.json.delay_encode(selected_endpoint_addr)`?

Reply:
selected_endpoint_addr is ONLY one endpoint by randomly selected.